app/components/pricing/calculator.tsx

```diff
@@ -143,14 +144,16 @@ export default function Calculator() {
 
         if (activeTab === "self-host") {
             if (mau >= 10000) {
```

Comment:
the limit here should not be 10k.. cause if someone has mfa checked, and they have 7k users, that will be 7k*0.02 which is more than 100... so it should be 5k if mfa is checked, else it should be 10k.

Reply:
done